packages/components/src/utils/hooks/useColorScheme.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  ColorSchemeName,
+  useColorScheme as RNUseColorScheme,
+} from 'react-native'
+
+/** Handles return of color scheme based on platform */
+export function useColorScheme(): ColorSchemeName {
+  // If not web Storybook, set with RN useColorScheme hook
+  if (!process.env.STORYBOOK_WEB) {
+    return RNUseColorScheme()
+  } else {
+    try {
+      const webStorybookColorScheme =
+        require('./useWebStorybookColorScheme').useWebStorybookColorScheme // eslint-disable-line
```

Comment:
Only functional change moving the hooks to their own files was updating this to point to the new location and renamed-to-fit-hook-naming-convention of starting with `use`.